e2e/utils.ts

```diff
@@ -72,10 +72,9 @@ export async function runCreateLlama(
   port: number,
   externalPort: number,
   postInstallAction: TemplatePostInstallAction,
+  llamaCloudProjectName: string,
+  llamaCloudIndexName: string,
 ): Promise<CreateLlamaResult> {
-  if (!process.env.OPENAI_API_KEY) {
```

Comment:
why not adding a check for `LLAMA_CLOUD_API_KEY` also instead?
the tests are not defined without these keys set